app/src/main/res/layout/activity_search.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/search_toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:theme="@style/ToolbarStyle"
+        app:title="@string/search_screen_title"
+        app:titleTextColor="@android:color/white">
+    </android.support.v7.widget.Toolbar>
```

Comment:
Let's nest this in an `AppBarLayout`, because I think that's a thing you're supposed to do because reasons.